app/views/admins/_user_entry.haml

```diff
@@ -1,131 +1,89 @@
 %li.user.media
   .media-left
-    - if user.person
-      .media-object
-        = person_image_tag(user.person, size: :thumb_small)
+    .media-object
+      = person_image_tag(person, size: :thumb_small)
   .media-body
     .row
       .col-sm-7
         %h4.media-heading
-          = user.try(:person).try(:name)
+          = person.diaspora_handle
+        = t(".remote") if person.remote?
+        = t(".local") if person.local?
+        %br
+        - if person.closed_account?
+          .label.label-danger
+            Closed
+        - else 
+          -if person.locked_access?
```

Comment:
- Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.
- The - symbol should have one space separating it from code